libs/connector/src/secretarium.connector.ts

```diff
@@ -190,7 +190,7 @@ export class SCP {
         return this._socket?.bufferedAmount || 0;
     }
 
-    connect(url: string, userKey: Key, knownTrustedKey: Uint8Array | string | undefined = undefined, protocol: NNG.Protocol = NNG.Protocol.pair1): Promise<void> {
+    connect(url = 'wss://on.klave.network', userKey: Key | undefined = undefined, knownTrustedKey: Uint8Array | string | undefined = undefined, protocol: NNG.Protocol = NNG.Protocol.pair1): Promise<void> {
```

Comment:
For the default values, I'd rather this be places with the rest of other constants in the `secretarium.constants.ts` file. But as my next comment describes we might not want to be doing this until SCP v2 is a reality